dynawo/sources/Modeler/DataInterface/IIDM/DYNStaticVarCompensatorInterfaceIIDM.cpp

```diff
@@ -64,8 +64,8 @@ StaticVarCompensatorInterfaceIIDM::getComponentVarIndex(const std::string& varNa
 
 void
 StaticVarCompensatorInterfaceIIDM::exportStateVariablesUnitComponent() {
-  staticVarCompensatorIIDM_.p(-1 * getValue<double>(VAR_P)* SNREF);
-  staticVarCompensatorIIDM_.q(-1 * getValue<double>(VAR_Q)* SNREF);
+  staticVarCompensatorIIDM_.p(getValue<double>(VAR_P)* SNREF);
+  staticVarCompensatorIIDM_.q(getValue<double>(VAR_Q)* SNREF);
```

Comment:
I have to admit that I don't remember how the overall mechanism works but is this change neutral for Modelica models?

Reply:
> I have to admit that I don't remember how the overall mechanism works but is this change neutral for Modelica models?

Thanks a lot for the review. You are absolutely right! I fixed it in my review commit, as well as the other things you pointed out.